prometheus_django/metrics.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from django.http import HttpResponse
+from prometheus_client import multiprocess
+from prometheus_client import generate_latest, CollectorRegistry, CONTENT_TYPE_LATEST
+
+
+# Create your views here.
```

Comment:
Are we supposed to copy paste this metrics method to our current microservices in the views.py file ? or this is going to be pointed someway by `urls.py ` ? 

This comment confuses me.

Reply:
If you see usage here: https://github.com/networklocum/application-service/pull/173/files it's just pointed in urls.py

Reply:
Based on https://github.com/networklocum/application-service/pull/173/files, I assume is the second (urls) which makes sense, then we should just remove the comment

Reply:
The comment "#Create your views here." ?

Reply:
Yes, that one. Sounds like if you have to copy your views into this file for tracking prometheus or something

Reply:
gotcha